app/components/primer/alpha/action_list/item.rb

```diff
@@ -48,7 +48,7 @@ class Item < Primer::Component
             Primer::BaseComponent.new(tag: :svg, width: "16", height: "16", **system_arguments)
           },
           content: lambda { |**system_arguments|
-            Primer::BaseComponent.new(tag: :span, **system_arguments)
+            Primer::BaseComponent.new(tag: :button, **system_arguments)
```

Comment:
Is this change necessary? The user could easily provide a button of their own here.